ts/features/itwallet/machine/eid/context.ts

```diff
@@ -7,6 +7,7 @@ import { IssuanceFailure } from "./failure";
 
 export type IdentificationContext =
   | { mode: "cieId"; level: "L2" }
+  | { mode: "cieIdL3"; level: "L3" }
```

Comment:
I see some redundancy here, could we use the level instead?
To me adding another mode introduces complexity without providing clear benefits. What do you think?
```suggestion
  | { mode: "cieId"; level: "L2" | "L3" }
```

Reply:
Thanks @mastro993, I simplified the flow using `context.isL3FeaturesEnabled`. BTW, the cieID L3 flow is still missing since we have to discuss some edge cases I found during the development of this PR.